sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -21,8 +21,7 @@ public ConversationAnalysisClientLiveTests(bool isAsync, ConversationAnalysisCli
         private static List<string> ExpectedOutput = new List<string>()
         {
             "2 plates",
-            "seared salmon nigiri",
-            "2 plates of seared salmon nigiri"
+            "seared salmon nigiri"
```

Comment:
Tip: I missed this before, but not worth holding up the PR: I recommend adding a terminal comma to object and collection initializers. It means the next time you add a line, only that new line shows up instead of a difference in the previous line to add a comma. That can also help mitigate some merge conflicts.